eth/eventdispatcher/event_dispatcher.go

```diff
@@ -73,6 +73,7 @@ func (ed *EventDispatcher) Start(ctx context.Context, fromBlock uint64) error {
 	if err != nil {
 		return fmt.Errorf("handle historical block events: %w", err)
 	}
+	ed.metrics.HistoricalLogsProcessed(lastProcessedBlock-fromBlock)
```

Comment:
I implemented `EventProcessed` and `EventProcessingFailed` metrics that track count of successful and failed events, so I think we don't need reporting how many blocks we processed, last processed block should be enough. I did it after you did your metrics, so sorry for not syncing with you on that in advance